build.sbt

```
@@ -35,10 +35,16 @@ lazy val commonSettings = Seq(
   }
 )
 
-lazy val fatalWarningsSettings = Seq(
+lazy val disableFatalWarnings = Seq(
```

```suggestion
lazy val fatalWarningsSettings = Seq(
```
Since this still enables fatal warnings (unless told not to), I think this should keep the old name.